app/src/main/java/chat/rocket/android/main/presentation/MainPresenter.kt

```diff
@@ -162,6 +164,14 @@ class MainPresenter @Inject constructor(
         }
     }
 
+    fun showLogoutDialog(context: Context) {
+        val builder = AlertDialog.Builder(context)
```

Comment:
The presenter doesn't have the responsibility to show that dialog. You should move this code to the ui.

Reply:
Okay